src/Shell/Task/ModelTask.php

```diff
@@ -875,6 +875,16 @@ public function listUnskipped()
     }
 
     /**
+     * Models never have routing prefixes applied.
+     *
+     * @return string
+     */
+    protected function _getPrefix()
```

Comment:
I can't follow?!
Example: ```bake all articles --routing-prefix admin --table-prefix wp_``` ?
Which context? Did this possibly become a bug because neither CLI flag nor matching protected method carried its context?